src/test/groovy/com/netgrif/application/engine/petrinet/service/PetriNetServiceTest.groovy

```diff
@@ -160,4 +168,51 @@ class PetriNetServiceTest {
         List<PetriNet> petriNets = petriNetService.findAllByUriNodeId(myNode.id)
         assert petriNets.size() == 2
     }
+
+    @Test
+    void processSearch() {
```

Comment:
All searches use only one query param at a time, it would be better to also test request with multiple params (eg. author + version + identifier)

Reply:
done